docs/ispc.rst

```diff
@@ -1721,6 +1722,7 @@ basic types:
 * ``int``: 32-bit signed integer; may also be specified as ``int32``.
 * ``unsigned int``: 32-bit unsigned integer; may also be specified as
   ``unsigned int32``, ``uint32`` or ``uint``.
+* ``float16``: 16-bit floating point value
```

Comment:
`fp16` suffix and how `float16` literals are supposed to be written needs to be documented.

Reply:
Added to doc